pkg/endpoints/utils.go

```diff
@@ -107,3 +112,45 @@ func (r Resource) RegisterReadinessProbes(container *restful.Container) {
 
 	container.Add(wsv4)
 }
+
+// Store all types here that are reused throughout files
+type Extension struct {
+	Name string
+	Url  string
+	Port string
+}
+
+
+func (r Resource) RegisterExtension(container *restful.Container, namespace string) {
+	logging.Log.Info("Adding API for extensions")
+	svcs, err := r.K8sClient.CoreV1().Services(namespace).List(metav1.ListOptions{LabelSelector: "tekton-dashboard-extension=true"})
+	if err != nil {
+		logging.Log.Debug("no extension found")
+		return
+	}
+	ws := new(restful.WebService)
+	ws.
+		Path("/").
+		Consumes(restful.MIME_JSON).
+		Produces(restful.MIME_JSON)
+	for _, svc := range svcs.Items {
```

Comment:
Big nit but let's newline this (I think just on saving in vscode you'd get it anyway)